### DISCUSSION TITLE
How to apply multiple formats to `content-hidden` and `content-visible`?

### DISCUSSION THREAD

**Discussion by cooknl** [quarto-dev/quarto-cli#2704] [Q&A]
It's not clear from the docs how to apply multiple formats to conditional content

<https://quarto.org/docs/authoring/conditional.html>

I don't know enough lua to know how to format multiple formats (list? array?)

<https://github.com/quarto-dev/quarto-cli/blob/346d80e55abc83e0a5b87e93a17730eac5656a92/src/resources/filters/quarto-pre/content-hidden.lua>

**Comment by cderv** [comment]
I don't think we handle several format to be passed. We have added `content-hidden` and `content-visible`, with both `unless-format` and `when-format` to target different scenario. 
Also we have added alias keyword to match several formats at once: https://quarto.org/docs/authoring/conditional.html#format-matching

Which cases are you trying to cover ? 
We may need to add alias or indeed try to make the feature works with several formats.

**Comment by cooknl** [comment]
docx & PDF both

NLC

Christophe Dervieux ***@***.***> schrieb am Mo., 3. Okt.
2022, 09:01:

**Reply by cscheid** [reply]
It should be possible to target them by repeating the condition (if it's not working, it's a bug on my code.)

```
::: {.content-hidden when-format="html" when-format="pdf"}

Should not appear in HTML or PDF.

:::
```

**Reply by cderv** [reply]
Thanks @cscheid for chiming in ! that was the thing I wanted to check next.

**Comment by cooknl** [comment]
I had considered that, but didn't try

I did try making a lua array


when-format= {"docx","pdf"}

But that didn't work


NLC

Christophe Dervieux ***@***.***> schrieb am Mo., 3. Okt.
2022, 16:32:
